planner/cascades/testdata/transformation_rules_suite_out.json

```diff
@@ -174,9 +174,9 @@
           "    TableGather_5 input:[Group#4]",
           "Group#4 Schema:[test.t.a,test.t.b,test.t.c], UniqueKey:[test.t.a]",
           "    TableScan_4 table:t, pk col:test.t.a",
-          "Group#3 Schema:[Column#14,Column#16,test.t.a]",
+          "Group#3 Schema:[Column#14,Column#16,test.t.a], UniqueKey:[test.t.a]",
           "    TableGather_5 input:[Group#5]",
-          "Group#5 Schema:[Column#14,Column#16,test.t.a]",
+          "Group#5 Schema:[Column#14,Column#16,test.t.a], UniqueKey:[test.t.a]",
```

Comment:
I think this is not right? PartialMode Agg's output cannot make the group by key as unique key.